Sources/TuistLoader/Up/UpMint.swift

```diff
@@ -0,0 +1,107 @@
+import Basic
+import Foundation
+import TuistSupport
+
+/// Command that installs Mint and packages.
+class UpMint: Up, GraphInitiatable {
+    /// A Boolean value indicating whether installing the packages of the Mintfile globally.
+    let linkPackagesGlobally: Bool
+
+    /// Up homebrew for installing Mint.
+    let upHomebrew: Upping
+
+    init(linkPackagesGlobally: Bool, upHomebrew: Upping = UpHomebrew(packages: ["mint"])) {
+        self.linkPackagesGlobally = linkPackagesGlobally
+        self.upHomebrew = upHomebrew
+        super.init(name: "Mint")
+    }
+
+    /// Default constructor of entities that are part of the manifest.
+    ///
+    /// - Parameters:
+    ///   - dictionary: Dictionary with the object representation.
+    ///   - projectPath: Absolute path to the folder that contains the manifest.
+    ///     This is useful to obtain absolute paths from the relative paths provided in the manifest by the user.
+    /// - Throws: A decoding error if an expected property is missing or has an invalid value.
+    required convenience init(dictionary: JSON, projectPath _: AbsolutePath) throws {
+        let linkPackagesGlobally: Bool = try dictionary.get("linkPackagesGlobally")
+        self.init(linkPackagesGlobally: linkPackagesGlobally)
+    }
+
+    /// Returns true when the command doesn't need to be run.
+    ///
+    /// - Parameters
+    ///   - projectPath: Path to the directory that contains the project manifest.
+    /// - Returns: True if the command doesn't need to be run.
+    /// - Throws: An error if the check fails.
+    override func isMet(projectPath: AbsolutePath) throws -> Bool {
+        if try !upHomebrew.isMet(projectPath: projectPath) {
+            return false
+        }
+
+        let mintfile = projectPath.appending(component: "Mintfile")
+        if !FileHandler.shared.exists(mintfile) {
+            throw MintError.mintfileNotFound(projectPath)
+        }
+
+        let output = try System.shared.capture(["cat", "\(mintfile.pathString)"])
```

Comment:
I'd use [this one](https://github.com/tuist/tuist/blob/master/Sources/TuistSupport/Utils/FileHandler.swift#L77) instead because it provides error handling out of the box.